bigquery/google/cloud/bigquery/query.py

```diff
@@ -31,7 +31,6 @@ class _SyncQueryConfiguration(object):
     Values which are ``None`` -> server defaults.
     """
     _default_dataset = None
-    _dry_run = None
```

Comment:
OK, I put that back(https://github.com/GoogleCloudPlatform/google-cloud-python/pull/2984/commits/63044573b771ab751302b934dae5d577c285786e).

The example request JSON in the docs you linked doesn't show `dryRun` on the query object AFAICT.